tests/framework/PostProcessors/TemporalDataMiningPostProcessor/DimensionalityReduction/test_tdmSKLearnLocallyLinearEmbedding.xml

```diff
@@ -86,7 +86,7 @@
         <plot>
           <type>line</type>
           <x>mData|Output|Time</x>
-          <y>mData|Output|PCAEmbeddingVector2</y>

```

Comment:
can't we plot the PCA components anymore?

Reply:
They at least have different names if they exist.

Reply:
Looking at the CSV files, they are called 

PCADimension1 | PCADimension2

Reply:
Fixed.